x-pack/plugins/fleet/server/routes/index.ts

```diff
@@ -20,3 +20,4 @@ export { registerRoutes as registerOutputRoutes } from './output';
 export { registerRoutes as registerSettingsRoutes } from './settings';
 export { registerRoutes as registerAppRoutes } from './app';
 export { registerLimitedConcurrencyRoutes } from './limited_concurrency';
+export { registerRoutes as registerPreconfigRoutes } from './policy_preconfig';
```

Comment:
nit: can we call this `registerPreconfigurationRoutes` instead?

(we've had some naming history with the word "config", so I'd prefer to use the full word "preconfiguration" even though it's quite long 🙂)